packages/mobile/src/escrow/EscrowedPaymentLineItem.tsx

```diff
@@ -2,18 +2,19 @@ import fontStyles from '@celo/react-components/styles/fonts'
 import * as React from 'react'
 import { StyleSheet, Text } from 'react-native'
 import { EscrowedPayment } from 'src/escrow/actions'
+import i18n from 'src/i18n'
 import { divideByWei, getCentAwareMoneyDisplay } from 'src/utils/formatting'
 
 interface Props {
   payment: EscrowedPayment
 }
 
 export default function EscrowedPaymentLineItem(props: Props) {
-  const { amount, message, recipientPhone } = props.payment
+  const { amount, recipientPhone } = props.payment
   return (
     <Text numberOfLines={1} ellipsizeMode="middle" style={styles.oneLine}>
       <Text style={[fontStyles.subSmall]}>
-        {recipientPhone} - {message}
+        {recipientPhone ? recipientPhone : i18n.t('global:unknown')} {i18n.t('global:for')}
       </Text>
       <Text style={[fontStyles.subSmall, fontStyles.semiBold]}>
         {' '}
```

Comment:
Concatenating strings like this makes it impossible to translate correctly. Some locale don't use the same ordering in words.
We could instead use the [`<Trans />`](https://react.i18next.com/latest/trans-component) component and [context](https://www.i18next.com/translation-function/context).

Something along those lines:
```ts
<Trans
  i18nKey="escrowPaymentLine"
  tOptions={{ context: !recipientPhone ? 'missingRecipientPhone' : null }}
>
  <Text numberOfLines={1} ellipsizeMode="middle" style={styles.oneLine}>
    <Text style={[fontStyles.subSmall]}>{{ recipientPhone }} for </Text>
    <Text style={[fontStyles.subSmall, fontStyles.semiBold]}>{{ moneyAmount }}</Text>
  </Text>
</Trans>
```

With the following translation strings (not sure about the correctness of the indices I used though):
```
"escrowPaymentLine": "<1><2>{{recipientPhone}} for</2><3>{{moneyAmount}}</3></1>",
"escrowPaymentLine_missingRecipientPhone": "<1><2>Unknown for</2><3>{{moneyAmount}}</3></1>",
```

Reply:
thanks a lot! It does make a lot of sense! I remember you've mention something like this on one of our meetings.